lusidtools/cocoon/utilities.py

```diff
@@ -915,3 +915,11 @@ def validate_mapping_file_structure(mapping: dict, columns: list, file_type: str
         raise ValueError(f"'required' mapping field not provided in mapping")
 
     pass
+
+
+def strip_whitespace(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Can you please add a docstring explaining what this function does and describing the parameters and return type?